interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/customize-appearance/report-visual-styles.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Report Visual Styles
+owner: Mary Sammal
+---
+# Report Visual Styles
+
+This topic describes how to combine [appearance properties](appearance-properties.md) into styles and apply them to report elements.
+
+## Create a Report Style
+
+Use the following approaches to create a visual style in your report:
+
+- **Extract a Style**
+
+    Specify [appearance properties](appearance-properties.md) for a report control and press **Extract Style** in the report's [toolbar](..\report-designer-tools\toolbar.md).
+
+    ![eurd-win-specify-properties](../../../../images/eurd-win-specify-properties.png)
+
+
+- **Create a new Style**
+
+    Press the caption button on the toolbar's Styles group to invoke the Styles Editor.
+
+    ![eurd-win-styles-add-styles](../../../../images/eurd-win-styles-add-style.png)
+
+    Press the **Add a style** button and specify appearance properties for the newly created style. Close the Styles Editor.
```

Comment:
Press the **Add a style** button and specify appearance properties for the newly created style. 
->
Press the **Add a style** button and specify the new style's appearance properties.

Reply:
ok